tests_page_object_pattern/test_dell_po.py

```diff
@@ -0,0 +1,33 @@
+from selene import browser, by, be, have
+from selene import config
+import pytest
+
+from rozetka_home_page import RozetkaHomePage
+from rozetka_search_results_page import RozetkaSearchResultsPage
+from products_open import ProductsOpen
+from basket_modal import BasketModal
+
+def test_rozetka_search_po():
+    config.timeout = 40
+# Open http://rozetka.com.ua home page
+    home_page = RozetkaHomePage()
+    home_page.open()
+# Search for ‘dell’
+    results_page = home_page.search("Dell")
+# Verify that at least 10 result links are displayed
+    results_page.verify_search_results_at_least(10)
+# Save the name and the price of 5th product, add it to the basket
+    pdp_results_page = ProductsOpen()
```

Comment:
the only time you need to explicitly create a new page object in test is the first page object